README.md

```diff
@@ -1 +1,157 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello-Microverse</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** This project is a Hello Microverse repository. No complex coding is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>front end</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+- added index.html 
+- added style.css
+
+- **[Display Hello-Microverse]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Browser
+- IDE(e.g.: Vscode,...)
+- Terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/Young-Gaius/Hello-Microverse
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd mv_hello-microverse
+
+  npm install or npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Open the live server or Open the index.html file in your browser
+
+```
+
+### Run Linters tests
+
+To run tests, run the following command:
+
+```sh
+  npx eslint . 
+  
+  npx stylelint "**/*.{css,scss}" --fix
+
+  ...
```

Comment:
- [ ] It seems 👀 that you didn't close the code comment in the correct place, please 🙏🏽 use ``` instead of `...` to be similar to the README template and for professional documentation.

![image](https://github.com/Young-Gaius/Hello-Microverse/assets/58642949/65701321-fffe-46be-bf21-b3bd6338c6bb)


In the first project, we must learn about:
   - Use the correct GitHub flow.
   - Professional documentation by using the README template and writing a descriptive PR.
   - Install liners.
If any of these points weren't present in future projects,  the PR will be invalidated.